docs/ingestion/kafka-ingestion.md

```diff
@@ -413,10 +421,13 @@ This query returns:
 |------------------|-----------|-----------------|-------------|
 |`development`|`wiki-edit`|`1680795276351`|`wiki-edits`|
 
-### Tuning configuration
+### `tuningConfig`
+
+Specify tuning properties in a `tuningConfig` object at the top level of an ingestion spec.
+
+The following table outlines the `tuningConfig` configuration properties specific to Apache Kafka.
 
-The following table outlines the `tuningConfig` configuration properties specific to Kafka.
-For configuration properties shared across all streaming ingestion methods, refer to [Supervisor tuning configuration](supervisor.md#tuning-configuration).
+For configuration properties shared across all supervisor-led ingestion, refer to [Supervisor `tuningConfig`](supervisor.md#tuningconfig).
```

Comment:
Why did you choose to replace "steaming ingestion methods" with "supervisor-led ingestion"?